portfolio/src/main/webapp/js/comments.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Gets comments from the '/comments' endpoint
+ * and displays them in the comments section.
+ */
+function loadComments() {
+  fetch('/comments')
+    .then(resp => resp.json())
+    .then(comments => {
+      console.log("Received comments: ");
+      console.log(comments);
+
+      removeCommentsOnPage();
+      comments.forEach((comment) => addCommentToPage(comment));  
+    })
+    .catch(err => {
+      console.error(err);
+      alert("Failed to populate comments.");
+    });
+}
+
+/**
+ * Removes the comments currently displaying on the page.
+ */
+function removeCommentsOnPage() {
+  const commentContainer = document.getElementById("comment-container");
+
+  let lastComment = commentContainer.lastElementChild;
+  while (lastComment) {
+    commentContainer.removeChild(lastComment);
+    lastComment = commentContainer.lastElementChild;
+  }
+}
+
+/**
+ * Adds a comment to the comments section UI.
+ * @param {{author: string, commentBody: string}} comment
+ */
+function addCommentToPage(comment) {
+  const commentContainer = document.getElementById("comment-container");
+  
+  // Create a new comment element
```

Comment:
This comment is redundant, `const newComment = document.createElement("div");` is self-explanatory.